bloomrun.js

```diff
@@ -29,7 +29,7 @@ function removePattern (bucket, pattern, payload) {
 
   for (var i = 0; i < bucket.data.length; i++) {
     if (pattern === bucket.data[i].pattern) {
```

Comment:
probably we should use https://github.com/mcollina/bloomrun/blob/master/lib/iterator.js#L5-L23 instead of `===`. What do you think?